shared/params/const.go

```diff
@@ -0,0 +1,11 @@
+package params
```

Comment:
I'd add comments for each one of them, what is it used?.. etc

Reply:
added